src/codegen/languages/javascript.ts

```diff
@@ -266,12 +266,8 @@ class JavaScriptFormatter {
   }
 }
 
-function quote(text: string, char: string = '\'') {
-  if (char === '\'')
-    return char + text.replace(/[']/g, '\\\'') + char;
-  if (char === '"')
-    return char + text.replace(/["]/g, '\\"') + char;
-  if (char === '`')
-    return char + text.replace(/[`]/g, '\\`') + char;
-  throw new Error('Invalid escape char');
+const quoteChar = '\'';
+function quote(text: string): string {
```

Comment:
it should now be called 'escape'

Reply:
it is still somehow a quote function because it adds the quote char ' before and after the input text so I would prefer quote here.